charts/opensearch-dashboards/Chart.yaml

```
@@ -15,7 +15,7 @@ type: application
 # This is the chart version. This version number should be incremented each time you make changes
 # to the chart and its templates, including the app version.
 # Versions are expected to follow Semantic Versioning (https://semver.org/)
-version: 2.16.0
+version: 2.17.0
```

```suggestion
version: 2.16.1
```
Update version per @prudhvigodithi's [requested change](https://github.com/opensearch-project/helm-charts/pull/530#discussion_r1546765134)